File: PHP/card.php
<?php

require_once('payment.php');
class Card extends Payment{
    public $numberCard;
    public $dateCard;
    public $cvv;

    public function __construct($id, $numberCard, $dateCard, $cvv){
        parent::__construct($id);
        $this->numberCard = $numberCard;
        $this->dateCard = $dateCard;
        $this->cvv = $cvv;
    }
}
?>

File: Python/card.py
from payment import Payment

class Card(Payment):
    cardNumber = int
    cardDate = str
    cvv = int
    
    def __init__(self, amount, kind, cardNumber, cvv, cardDate) -> None:
        super().__init__(amount, kind)
        self.cardNumber = cardNumber
        self.cvv = cvv
        self.cardDate = cardDate